src/components/home/Main.tsx

```diff
@@ -2,17 +2,20 @@ import React from 'react'
 import { ProfileCard } from '@/components/home/ProfileCard'
 import classes from '@/components/home/style.css'
 import useGetUsers from '@/components/home/useGetUsers'
+import { useRecoilValue } from 'recoil'
+import { userInfoState } from '@/atoms/userInfoAtom'
 
 export const Main = () => {
   const { Users, transferToProfile } = useGetUsers()
```

Comment:
慣習では`const { Users, setUsers } = useGetUsers()`と書くのではなかったでしたっけ？

Reply:
この場合、custom hooksなので、何を返すかは任意(開発者次第)なので、問題ないと思います！
ただ、オブジェクトではなく配列で返して、
```suggestion
  const [ users, transferToProfile ] = useGetUsers()
```
の方がuseStateとはそろってるのかも？と思いました！
(custom hooksの場合にはオブジェクトを使うのが多いとかあったらすみません)

Reply:
@summerTakashi @kan-mayoshi 
真吉さん指摘の通り、custom hooksを定義しているので、問題ありません。
配列だと型エラーが出るが、オブジェクトだと出なかったので、オブジェクトを使いました。(Custom hooksの定義の問題？)